cmd/genji/insert.go

```diff
@@ -108,10 +110,20 @@ func executeInsertCommand(db *genji.DB, table string, r io.Reader) error {
 	return nil
 }
 
-func runInsertCommand(e, DBPath, table string, args []string) error {
+func runInsertCommand(e, DBPath, table string, auto bool, args []string) error {
 	var ng engine.Engine
 	var err error
 
+	ts := time.Now().Unix()
+	if DBPath == "" && auto {
+		DBPath = "data_" + strconv.FormatInt(ts, 10) + ".db"
```

Comment:
This only works with Bolt, because Badger creates a directory instead of a file.
We should take that into account when generating the path.

I think we should simply name the directory the same way, just without the ".db" suffix.